package.json

```diff
@@ -52,7 +52,7 @@
     "cy": "CYPRESS_dexUrl=https://$INGRESS_HOST:$PORT_HTTPS CYPRESS_baseUrl=http://localhost:9999 cypress open",
     "cy:dev": "source ../monitor-ci/.env && CYPRESS_dexUrl=CLOUD CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{cloud,shared}/**/*.*'",
     "cy:dev-oss": "source ../monitor-ci/.env && CYPRESS_dexUrl=OSS CYPRESS_baseUrl=https://$INGRESS_HOST:$PORT_HTTPS cypress open --config testFiles='{oss,shared}/**/*.*'",
-    "generate": "export SHA=8e8ba92e4764cd8914284315fb87c1945c757497 && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
+    "generate": "export SHA=973d50bee0aa6cad9c180995c9d27a5dc5d96c8c && export REMOTE=https://raw.githubusercontent.com/influxdata/openapi/${SHA}/ && yarn generate-meta",
```

Comment:
The PR with this SHA hasn't been merged yet and this looks like the SHA of your commit.  Looking at the last PR to go in, it looks like the PR itself gets a different commit SHA so I think it's probably best to wait until that PR has been merged and update this change to match the new SHA for openapi's master branch.

Reply:
that's correct. I had that pointed out and is on my radar already. but definitely a good catch. let me get the other one merged.

Reply:
Just merged and updated package.json